util/python/python_config.sh

```diff
@@ -46,7 +46,7 @@ function main {
 }
 
 function python_path {
-  python - <<END
+  $PYTHON_BIN_PATH - <<END
```

Comment:
Are you running this script directly? I encounter this issue when I run `./configure` and provide custom python path in when it asks for that. `configure` will invoke this `python_config.sh`.